src/main/kotlin/enums/Operator.kt

```diff
@@ -1,16 +1,28 @@
 package enums
 
-enum class Operator(val operator: String) {
-    PLUS("+"),
-    MINUS("-"),
-    MULTIPLY("*"),
-    DIVIDE("/");
+enum class Operator(val operator: String, val calculation: (Int, Int) -> Int) {
+    PLUS("+", { left, right -> left + right }),
+    MINUS("-", { left, right -> left - right }),
+    MULTIPLY("*", { left, right -> left * right }),
+    DIVIDE("/", {
+            left, right -> require(right != 0) { "${ErrorCode.ERROR_DIVIDER_CANNOT_ZERO.code }: 나누는 수는 0이 될 수 없습니다." }
+            left / right
+        }
+    );
 
     companion object {
-        fun getOperators(): List<String> {
-            return enumValues<Operator>().map {
-                it.operator
-            }.toList()
+        private val mapping = enumValues<Operator>().associate { it.operator to it.calculation }
```

Comment:
```suggestion
        private val mapping =  values().associateBy(Operator::symbol)
```

associateBy로 좀 더 간편하게 매핑이 가능합니다

Reply:
associateBy를 사용하면 객체 자체가 매핑이 되어서 associate로 operator -> 람다 함수 매핑되도록 작성했습니다.

틀린 부분 있다면 설명해주시면 감사하겠습니다 ㅎ.ㅎ

Reply:
enumValues() 부분은 values()로 변경하도록 하겠습니다.